code/core/constants.h

```diff
@@ -1,5 +1,6 @@
 #ifndef constants_h
 
 #define NLAYERS 6
+#define PTCUT 1.0f
```

Comment:
Initially I was confused about the inclusion of this file to other files but now I understand that this definition is needed. However, this is not the correct pT cut... That prompts me to ask a question larger than this PR but possibly connected to the alpaka migration:
Are most of the files in the `code/core` directory supposed to be a CPU version of our code? If so, isn't the CPU code outdated and also wrong (as details like the one that prompted this comment seem to imply)? It's also not used, as far as I know.
Should we remove it, given the CPU version coming automatically with the move to the alpaka framework? Maybe @sgnoohc is the best person to ask. We should possibly make an issue to discuss and understand if there are any action items here for a future PR.

Reply:
From the looks of it, I think there was a mistake when the cpu code was moved out of SDL? If you look, there are duplicate files in the cpu directory and code/core. So there are a lot of files in code/core that are just the old cpu files. My plan was to remove all of the old cpu code in a subsequent PR.